src/H5Apublic.h

```diff
@@ -94,8 +94,12 @@ H5_DLL herr_t H5Aclose(hid_t attr_id);
  * \ingroup ASYNC
  * \async_variant_of{H5Aclose}
  */
-H5_DLL herr_t H5Aclose_async(const char *app_file, const char *app_func, unsigned app_line, hid_t attr_id,
-                             hid_t es_id);
+H5_DLL herr_t H5Aclose_async(
+#ifndef H5_DOXYGEN
+                             const char *app_file, const char *app_func, unsigned app_line,
```

Comment:
This isn't going to compile because the closing parenthesis is inside the #else clause. But also, I don't think we can just do this because it's changing the signature of the API itself. All the parameters always need to be there.